pkg/service/roomservice.go

```diff
@@ -159,17 +122,10 @@ func (s *RoomService) DeleteRoom(ctx context.Context, req *livekit.DeleteRoomReq
 		return nil, err
 	}
 
-	if s.roomAllocator.CreateRoomEnabled() {
-		_, err := s.router.CreateRoom(ctx, &livekit.CreateRoomRequest{Name: req.Room})
-		if err != nil {
-			return nil, err
-		}
-	} else {
-		done, err := s.startRoom(ctx, livekit.RoomName(req.Room))
-		if err != nil {
-			return nil, err
-		}
-		defer done()
+	// ensure at least one node is available to handle the request
+	_, err = s.router.CreateRoom(ctx, &livekit.CreateRoomRequest{Name: req.Room})
```

Comment:
does this honor `AutoCreate`? it seems like we do not want to create the room record again right before deleting it, in most cases

Reply:
we check a few lines earlier that the room exists in the db. we're using the internal createroom api to start the room on a media node, not to create the room